icicle/tests/test_base.h

```diff
@@ -53,7 +49,13 @@ class IcicleTestBase : public ::testing::Test
   static void TearDownTestSuite() {}
 
   // SetUp/TearDown are called before and after each test
-  void SetUp() override {}
+  void SetUp() override
+  {
+    unsigned seed = time(NULL);
+    srand(seed);
+    ICICLE_LOG_INFO << "Seed for tests is: " << seed;
```

Comment:
tests -> test

Reply:
Can we have the init code run at default or at least create a macro for that

Reply:
What do you mean Mickey?

Reply:
every test should start with a copy of those 3 lines.
you can create a macro that will shrink it to 1 line or is there a code in test base that runs before every test? you can place it there

Reply:
// SetUp/TearDown are called before and after each test

This function is already the macro you're referring to

Reply:
disregard - all good